source/user-manual/agents/remote-upgrading/create-custom-wpk-automatically.rst

```diff
@@ -0,0 +1,106 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _create-custom-wpk-automatically:
+
+Automated custom WPK packages creation
+============================
+
+Wazuh provides an automated way of building WPK packages.
```

Comment:
Here, a larger description is needed. We can mention what is the process for the WPK building: the use of docker, the output, the automation capability, etc.

Reply:
I have added a bit longer description in https://github.com/wazuh/wazuh-documentation/pull/1308/commits/f1780dd4ad9aa2688d10d80739e742caedb4139e